pkg/tests/handlers/accounts.go

```diff
@@ -226,12 +227,23 @@ func TestClusterAuthorizations(ctx context.Context, options *types.TestOptions)
 	return nil
 }
 
+func randomizeResourceName(ResourceName_list [5]string) string {
```

Comment:
For naming conventions GO uses MixedCaps, and the leading letter should be only capital if it is going to be Publicly accessible.
A function parameter should not be constrained to a slice size.
Also, if you name the variable right there is no need to specify that it is a list.